src/main/java/seedu/address/AlfredException.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address;
+
+public abstract class AlfredException extends Exception {
+
+
+    public AlfredException(String message) {
+        super(message);
+    }
+
+    public AlfredException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+}
```

Comment:
Is there an extra closing brace here?

Reply:
What do you mean haha, i couldnt find the extra brace.

Reply:
Ah soz all my comments outdated liao lol, looks good